admin/src/collections/RecipientsRecentPayments.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+	AdditionalFieldDelegate,
+	AsyncPreviewComponent,
+	buildCollection,
+	buildProperties,
+	buildProperty,
+	StringPropertyPreview,
+} from '@camberi/firecms';
+import { Recipient, RECIPIENT_FIRESTORE_PATH } from '@socialincome/shared/src/types';
+import { getMonthIDs } from '@socialincome/shared/src/utils/date';
+import { CreateOrangeMoneyCSVAction } from '../actions/CreateOrangeMoneyCSVAction';
+import { BuildCollectionProps } from './index';
+import { paymentsCollection, paymentStatusMap } from './Payments';
+import { firstNameProperty, lastNameProperty, orangeMoneyUIDProperty, programStatusProperty } from './Recipients';
+
+let currentDate = new Date();
```

Comment:
While we are on it: shall we change it to `const`s?

Reply:
yes, sure👍🏼